osu.Game/Screens/Edit/Compose/Components/EditorSelectionHandler.cs

```diff
@@ -247,9 +247,7 @@ public void SetNewCombo(bool state)
         {
             EditorBeatmap.PerformOnSelection(h =>
             {
-                var comboInfo = h as IHasComboInformation;
-
-                if (comboInfo == null || comboInfo.NewCombo == state) return;
+                if (h is not IHasComboInformation comboInfo || comboInfo.NewCombo == state) return;
```

Comment:
dunno about this one, if using `is not` here i'd probably split the guards into two